### ISSUE TITLE
Option to allow non-indented comments?

### ISSUE THREAD

**Issue by emmericp** [CppCXY/EmmyLuaCodeStyle#162]
I have some code that contains a few commented out sections like this:

```
function foo(bar, baz)
    if foo then
--      if baz then
--          return baz
--      end
        return bar
    end
end
``` 

This was likely done by some editor function that just supports commenting out like this. It currently triggers a warning that indent is expected here. It would be great to have an option that always allow comments that start at the beginning of the line (and keeps them there during formatting) instead of at the designated indention. But comments starting at a non-zero and unexpected indention level should still trigger warnings.

**Comment by CppCXY** [comment]
It's Chinese New Year soon, so I'll probably try it after the holidays.

**Comment by emmericp** [comment]
thank you!
